### PR TITLE
Updates on the example's jupyter notebook and dataloader.py following the new #66 util.py

### PR DESCRIPTION
**Pull request by pavlos-p**
## Description
@rsandmann converted `models` into `self.models` on the example's Jupyter Notebook.

The example's Jupyter notebook and `dataloader.py` have been updated to follow #66 changes.

Merge after #66.

## How has this been tested?
- Tested the Jupyter Notebook after a fresh fork and bazel build.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by TTitcombe** [comment]
It looks as if the test failure is nothing to do with this PR. I've opened up an issue for this (and other randomly failing tests I've seen) - https://github.com/OpenMined/PyVertical/issues/65

**Comment by pavlos-p** [comment]
Tests randomly failing. Related Issue #65.

**Comment by TTitcombe** [comment]
@pavlos-p merge conflict

File: src/dataloader.py [hunk 1]
```diff
@@ -75,15 +75,15 @@ def __iter__(self):
     def __len__(self):
         return (len(self.dataloader1) + len(self.dataloader2)) // 2
 
-    def drop_non_intersecting(self, intersection1: List[int], intersection2: List[int]):
+    def drop_non_intersecting(self, intersection: List[int]):
```
Comment (TTitcombe) [inline src/dataloader.py:78]: the tests for `drop_non_intersecting` will need to be changed

**Review by TTitcombe — CHANGES_REQUESTED** [review]

File: src/dataloader.py (hunk 1 above)
Comment (pavlos-p) [reply src/dataloader.py:78]: Changed the tests, and commented out the last one `test_datasets_have_same_ids_after_drop_non_intersecting()`. But, they are still failing. In my local version, using  `python -m pytest tests/`, I get `26 passed, 1 warning in 44.57s`.

File: tests/test_dataloader.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+import sys
+sys.path.append('../')
```
Comment (TTitcombe) [inline tests/test_dataloader.py:2]: Can we move the import and `sys.path.append` to after the docstring

File: tests/test_dataloader.py [hunk 3]
```diff
@@ -89,26 +93,38 @@ def test_drop_non_intersecting_removes_all_elements_with_empty_intersection(self
...
+        dataloader.drop_non_intersecting(intersection)
 
         assert len(dataloader.dataloader1.dataset.data) == 0
         assert len(dataloader.dataloader1.dataset.ids) == 0
         assert len(dataloader.dataloader2.dataset.targets) == 0
         assert len(dataloader.dataloader2.dataset.ids) == 0
 
-    def test_datasets_have_same_ids_after_drop_non_intersecting(self):
-        dataloader = VerticalDataLoader(self.dataset, batch_size=128)
+    # def test_datasets_have_same_ids_after_drop_non_intersecting(self):
```
Comment (TTitcombe) [inline tests/test_dataloader.py:103]: Why was this test commented out? If we still want to keep it to uncomment it later, can we instead remove the `#` and rename it to `def xtest.....` the `x` at the start of the function name means `pytest` ignores this test

**Review by TTitcombe — CHANGES_REQUESTED** [review]

**Comment by TTitcombe** [comment]
LGTM. Thanks @pavlos-p
